### ISSUE TITLE
Error 500 crash when creating a data trigger for a datastream object interface

### ISSUE THREAD

**Issue by davidebriani** [astarte-platform/astarte#523]
Trying to create a data trigger for a Datastream Object interface results in a 500 `Internal Server Error` from APIs.

Trigger source:
```json
{
    "name": "trigger_for_datastream_object",
    "action": {
        "http_url": "http://domain.com/webhook",
        "ignore_ssl_errors": false,
        "http_method": "post",
        "http_static_headers": {}
    },
    "simple_triggers": [
        {
            "type": "data_trigger",
            "interface_name": "my.simple.DatastreamObject",
            "interface_major": 0,
            "on": "incoming_data",
            "match_path": "/simple/endpoint",
            "value_match_operator": ">",
            "known_value": 0
        }
    ]
}
```

Related interface:
```json
{
    "interface_name": "my.simple.DatastreamObject",
    "version_major": 0,
    "version_minor": 1,
    "type": "datastream",
    "ownership": "device",
    "aggregation": "object",
    "mappings": [
        {
            "endpoint": "/simple/endpoint",
            "type": "double"
        }
    ]
}
```

**Comment by rbino** [comment]
This happens because Realm Management replies with a generic error with name `invalid_object_aggregation_trigger`, but this is not handled [here](https://github.com/astarte-platform/astarte/blob/master/apps/astarte_realm_management_api/lib/astarte_realm_management_api_web/controllers/fallback_controller.ex#L83) so the conversion to existing atom fails.

A clause handling that error (and the relative JSON error template) should be added to `RealmManagement.APIWeb.FallbackController`.

**Comment by rbino** [comment]
Closed by #686
